Migrates workbox module docs from WebFu to developer.chrome.com (#1802)
GoogleChrome/developer.chrome.com: site/en/docs/workbox-modules/workbox-build/index.md

---
layout: 'layouts/doc-post.njk'
title: workbox-build
date: 2018-01-31
updated: 2020-01-17
description: >

>   An npm module that can generate a complete service worker, inject a precache manifest and copy the Workbox files.

Reviewer: ```suggestion
  A module that can generate a service worker, inject a precache manifest, or create a local copy the Workbox libraries.
```
